core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -4284,6 +4293,14 @@ private static Function<Page, Page> enforceLoadedLayoutProcessor(List<Symbol> ex
         return new PageChannelSelector(channels);
     }
 
+    private static Page enforceSpooledLayoutProcessor(Page page)
```

Comment:
enforce is wrong name. It suggest it does some action so the condition is met. Here we are just verifying current state.

Reply:
validate?